modules/sov-ibc-transfer/src/context.rs

```diff
@@ -203,25 +249,27 @@ where
         coin: &PrefixedCoin,
         memo: &Memo,
     ) -> Result<(), TokenTransferError> {
-        let token_id_buf = general_purpose::STANDARD_NO_PAD
-            .decode(memo.as_ref())
-            .map_err(|e| TokenTransferError::Other(format!("Failed to decode memo: {e}")))?;
+        if !memo.as_ref().is_empty() {
+            return Err(TokenTransferError::Other(
+                "Memo must be empty when escrowing tokens".to_string(),
+            ));
+        }
```

Comment:
Comments on memo being empty (for time being) here too.

Reply:
fyi, ended up allowing memos but with the same size limit that ibc-go uses.